### PR TITLE
[ja] Translate tasks/debug-application-cluster/debug-application into Japanese

### LINKED ISSUE #30873: [ja] Translate tasks/debug-application-cluster/debug-application into Japanese
# This is a Feature Request

## What would you like to be added

Translate [tasks/debug-application-cluster/debug-application](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-application/)  into Japanese.

## Why is this needed

This page has not been translated.

## Comments

/language ja
/assign

### PR DESCRIPTION
**Pull request by ptux**
resolve #30873

### CODE REVIEW

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:89]:
```suggestion
レプリケーションコントローラーはかなり単純なものです。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:91]:
```suggestion
もしPodを作成できないのであれば、[上記の説明](#debugging-pods)を参照して、Podをデバッグしてください。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:92]:
```suggestion
また、`kubectl describe rc ${CONTROLLER_NAME}`を使用すると、レプリケーションコントローラーに関連するイベントを確認することができます。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:96]:
```suggestion
Serviceは、Podの集合全体でロードバランシングを提供します。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:94]:
```suggestion
### Serviceのデバッグ
```

Podに対してServiceですね

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:97]:
```suggestion
Serviceが正しく動作しない原因には、いくつかの一般的な問題があります。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:99]:
```suggestion
以下の手順は、Serviceの問題をデバッグするのに役立つはずです。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:101]:
```suggestion
まず、Serviceに対応するEndpointが存在することを確認します。
```
この場合Endpointリソースを指していると思います

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:102]:
```suggestion
全てのServiceオブジェクトに対して、apiserverは `endpoints` リソースを利用できるようにします。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:109]:
```suggestion
EndpointがServiceのメンバーとして想定されるPod数と一致していることを確認してください。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:110]:
```suggestion
例えば、3つのレプリカを持つnginxコンテナ用のServiceであれば、ServiceのEndpointには3つの異なるIPアドレスが表示されるはずです。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:112]:
```suggestion
#### Serviceに対応するEndpointがありません
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:114]:
```suggestion
Endpointが見つからない場合は、Serviceが使用しているラベルを使用してPodをリストアップしてみてください。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:125]:
```suggestion
セレクタに一致するPodを一覧表示するには、次のコマンドを使用します。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:131]:
```suggestion
リストがServiceを提供する予定のPodと一致することを確認します。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:132]:
```suggestion
Podの`containerPort`がServiceの`targetPort`と一致することを確認します。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:141]:
```suggestion
[Debugging Service document](/docs/tasks/debug-application-cluster/debug-service/)で、`Service` が実行されていること、`Endpoints`があること、`Pods`が実際にサービスを提供していること、DNS が機能していること、IPtablesルールがインストールされていること、kube-proxyが誤作動を起こしていないようなことを確認してください。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:143]:
```suggestion
[トラブルシューティングドキュメント](/docs/tasks/debug-application-cluster/troubleshooting/)に詳細が記載されています。
```

**Review by nasa9084 — CHANGES_REQUESTED** [review]

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (riita10069) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:40]:
```suggestion
Podが`Pending`で止まっている場合、それはノードにスケジュールできないことを意味します。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (riita10069) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:47]:
```suggestion
* **リソースが不足しています。** クラスターのCPUまたはメモリーを使い果たしている可能性があります。Podを削除するか、リソースの要求値を調整するか、クラスターに新しいノードを追加する必要があります。詳しくは[Compute Resources document](/ja/docs/concepts/configuration/manage-resources-containers/)を参照してください。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (riita10069) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:54]:
```suggestion
Podが`Waiting`状態で止まっている場合、ワーカーノードにスケジュールされていますが、そのノード上で実行することができません。この場合も、`kubectl describe ...`の情報が参考になるはずです。`Waiting`状態のPodの最も一般的な原因は、コンテナイメージのプルに失敗することです。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (riita10069) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:59]: Isn't this the registry?
However, the upstream documentation describes it as a repository.
"Repository" is a collection of container images or other artifacts in the registry that have the same name but different tags.

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (riita10069) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:136]:
```suggestion
詳しくは[Serviceのデバッグ](/ja/docs/tasks/debug-application-cluster/debug-service/)を参照してください。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (riita10069) [reply content/ja/docs/tasks/debug-application-cluster/debug-application.md:141]:
```suggestion
[Serviceのデバッグに関するドキュメント](/ja/docs/tasks/debug-application-cluster/debug-service/)で、`Service` が実行されていること、`Endpoints`があること、`Pods`が実際にサービスを提供していること、DNS が機能していること、IPtablesルールがインストールされていること、kube-proxyが誤作動を起こしていないようなことを確認してください。
```

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (ptux) [reply content/ja/docs/tasks/debug-application-cluster/debug-application.md:57]: I think you are right.
And I have created PR for the upstream.

> https://github.com/kubernetes/website/pull/31170

Let's wait until the PR merged.

Comment (ptux) [reply content/ja/docs/tasks/debug-application-cluster/debug-application.md]: Got #31170 PR merged.
And changed `リポジトリ` to `レジストリ`.

File: content/ja/docs/tasks/debug-application-cluster/debug-application.md
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/debug-application.md:17]:
```suggestion
Podなのか、レプリケーションコントローラーなのか、それともサービスなのか？
```

**Review by nasa9084 — CHANGES_REQUESTED** [review]

**Comment by ptux** [comment]
@nasa9084 @inductor 

You'd better merge the PR with squash.

`/label tide/merge-method-squash`

**Pull request merged**
